mackerel-plugin-aws-kinesis-streams/lib/aws-kinesis-streams.go

```diff
@@ -0,0 +1,245 @@
+package mpawskinesisstreams
+
+import (
+	"errors"
+	"flag"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatch"
+	mp "github.com/mackerelio/go-mackerel-plugin-helper"
+)
+
+const (
+	namespace          = "AWS/Kinesis"
+	metricsTypeAverage = "Average"
+	metricsTypeSum     = "Sum"
```

Comment:
At first I used `metricsTypeSum`, but afterwards I changed them to Average. (ref: https://github.com/astj/mackerel-plugin-aws-kinesis-streams/commit/389cedb772d3678964f5525f38406135d00e03fe)
So I'll remove it thanks.